src/logger/logger.ts

```diff
@@ -4,6 +4,13 @@ import * as uuid from 'uuid';
 import stringify from 'fast-safe-stringify';
 import isError from 'is-error';
 
+export const redactSecret = (data: any): any => {
+  return data.replace(
+    /(\\*"client_secret\\*":\\*")([a-zA-Z0-9_+=.@/-]{60,})(\\*")/gi,
```

Comment:
I'd probably be less restrictive on the format of `client_secret` - I think we should redact any value even if it's not matching the regex as we don't really control what characters this may be made of

Reply:
so you suggest any characters except " or '  right?

Reply:
Yes, or even any value in the `client_secret` property

Reply:
that can be problematic, because "client_secret" is how we identify the secret, otherwise we can only check for a word with 64 characters and that can mach a lot of things, like links in stack-trace etc...
it does not have nice format like token ...

Reply:
Can we match e.g. anything that's between "client_secret:" and the next comma or quote in the string? My worry is that these are generated by another team and once they decide to make them 63 characters long instead of 64 this may break

Reply:
now we're matching most of the characters [a-zA-Z0-9_+=.@/-] and length is 60+ 
in next commit I will match all characters except " ' \
I will keep the length as 60+ or do you want to count with shorter secret?

Reply:
I think the length shouldn't matter because again - it's out of our control

Reply:
ok removed the length condition